content/influxdb/v2.0/reference/config-options.md

```diff
@@ -345,6 +346,59 @@ engine-path = "/users/user/.influxdbv2/engine"
 
 ---
 
+### flux-log-enabled
+
+Include option to show detailed logs for Flux queries, including the following information:
+
+- query response size (`response_size`)
+- query errors (`err`)
+- query durations (`stat_total_duration`, `stat_compile_duration`, `stat_execute_duration`)
+- memory allocation (`stat_max_allocated`, `stat_total_allocated`)
```

Comment:
@wbaker85 @russorat is these accurate and/or can we provide detail to clarify what's being logged? For example, a comprehensive list of errors? Which bits the duration and allocated are measuring?

Reply:
Here's what all the log fields mean:
- `compiler_type`: Compiler used for processing the query (will always be Flux)
- `response_size`: Size of the response, in bytes
- `query`: The textual representation of the query
- `err`: Any errors encountered while processing the query
- `stat_total_duration`: Total duration to process the query
- `stat_compile_duration`: Duration to compile the query
- `stat_execute_duration`: Duration to execute the query
- `stat_max_allocated`: Maximum amount of memory allocated while processing the query, in bytes
- `stat_total_allocated`: Total amount of memory allocated while processing the query, in bytes. This includes memory that was freed and then used again.

The list of potential errors would be vast I'd imagine, so I don't think we'd want to list them all here. I think it will be sufficient to describe it as just "any errors encountered" as I've got listed above.

Reply:
thank you, @wbaker85

Reply:
@wbaker85 are the durations in seconds?